Mathlib/Order/Interval/Finset/Nat.lean

```diff
@@ -67,6 +67,11 @@ theorem Iio_eq_range : Iio = range := by
 theorem Ico_zero_eq_range : Ico 0 = range := by rw [← Nat.bot_eq_zero, ← Iio_eq_Ico, Iio_eq_range]
 #align nat.Ico_zero_eq_range Nat.Ico_zero_eq_range
 
+lemma range_eq_Icc_zero_minus (n : ℕ) (hn : n > 0): range n = Icc 0 (n - 1) := by
```

Comment:
Please use `sub_one` or `pred` in the name. Also, please assume `n ≠ 0`, not `n > 0`. Otherwise LGTM. Thanks!

Reply:
Done, thanks.